components/resources/ai_chat_ui_strings.grdp

```diff
@@ -264,6 +264,27 @@
   <message name="IDS_CHAT_UI_LEARN_MORE" desc="Link text to learn more about a feature">
     Learn more
   </message>
+  <message name="IDS_CHAT_UI_CLOSE_NOTICE" desc="Label for button which closes a notice">
+    Close this notice
+  </message>
+  <message name="IDS_CHAT_UI_NOTICE_CONVERSATION_HISTORY_TITLE" desc="Title for the notice about conversation history">
+    Conversation history
+  </message>
+  <message name="IDS_CHAT_UI_NOTICE_CONVERSATION_HISTORY_BODY" desc="Main text for the notice about conversation history">
+    Leo will now remember your previous conversations so you can go back to them. They are stored privately on your device, and you can delete them any time.
+  </message>
+  <message name="IDS_CHAT_UI_NOTICE_CONVERSATION_HISTORY_EMPTY" desc="Main text for a notice that the conversation list is empty">
+    Your conversation history will appear here once you start a conversation.
+  </message>
+  <message name="IDS_CHAT_UI_NOTICE_CONVERSATION_HISTORY_TITLE_DISABLED_PREF" desc="Title for a notice that the conversation history preference is disabled">
+    History is disabled
+  </message>
+  <message name="IDS_CHAT_UI_NOTICE_CONVERSATION_HISTORY_DISABLED_PREF" desc="Main text for a notice that the conversation history preference is disabled">
+    In order to view and search your previous conversations with Leo, you need to enable conversation history.
```

Comment:
Similarly here, I don't think we need the word "previous". Not blocking.

Reply:
I'd argue previous in this case is necessary (although I'll remove the "and search" part because we don't have that). Feels a bit strange to be viewing a conversation (a non-stored one) with a notice saying "In order to view your conversations with Leo, you need to...".